src/Utility.java

```diff
@@ -15,9 +14,23 @@ protected static void displayMenu() {
         System.out.println("\t7. Exit");
     }
 
-    /* protected static void displayHelpMenu() {
+    protected static void displayHelpMenu() {
+        Scanner displayHelpMenuScanner = Scanner(System.in);
+        System.out.println("Help Menu:");
```

Comment:
you'll need to change line 18 to be
`Scanner displayHelpMenuScanner = new Scanner(System.in);`